wallets-testing/services/service.widget.ts

```diff
@@ -0,0 +1,102 @@
+import { ETHEREUM_WIDGET_CONFIG } from '../config';
+import { WalletTypes } from '@lidofinance/wallets-testing-wallets';
+import { Page, test, expect } from '@playwright/test';
+import { BrowserService } from '@lidofinance/browser-service';
+import { WidgetPage } from '../pages/widget.page';
+import { waitForTextContent } from '../utils/helpers';
+
+export interface TxConfig {
+  txAmount: string;
+  tokenAddress?: string;
```

Comment:
Could you check, looks like `tokenAddress` && `mappingSlot` redundant. It was used in old approach